### DISCUSSION TITLE
Kubernetes deployment through Helm chart

### DISCUSSION THREAD

**Discussion by zzzming** [langflow-ai/langflow#2078] [General]
Enterprise customers often use a helm chart for installation on kubernetes. We would like to upstream the helm chart for LangFlow in the repo. Do you have any suggestion? Can we submit the helm chart under the /deploy folder?

This is the helm chart
https://github.com/datastax/ragstack-ai-langflow/tree/ragstack-main/ragstack/deploy/kubernetes/helm/langflow

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
